tests/unit/ops/transformer/inference/inference_test_utils.py

```diff
@@ -26,12 +26,7 @@ def get_tolerances():
 def get_dtypes():
     global DTYPES
     if DTYPES is None:
-        DTYPES = [torch.float16, torch.float32]
-        try:
-            if get_accelerator().is_bf16_supported():
-                DTYPES.append(torch.bfloat16)
-        except (AssertionError, AttributeError):
-            pass
+        DTYPES = get_accelerator().supported_dtypes()
```

Comment:
Thanks for making this change. I checked and it seems that `torch.float32` may not be included in the `.supported_dtypes()` of the accelerators. Can you please double check that?

Reply:
Thanks for checking.